Source/Mosa.Compiler.MosaTypeSystem/SignatureComparer.cs

```diff
@@ -64,14 +64,14 @@ public static int GetHashCode(MosaType type)
 			// Hash code DOES not need to be unique, so to save time ArrayInfo is not hashed
 			case MosaTypeCode.Array:
 				if (type.ElementType == null)
-					throw new InvalidOperationException("Element type of type is null!");
+					throw new InvalidCompilerOperationException("Element type of type is null");
```

Comment:
Inconsistent removal of exclamation marks

Reply:
Found and updated.